src/main/java/org/moltimate/moltimatebackend/service/AlignmentService.java

```diff
@@ -64,97 +64,68 @@ public ActiveSiteAlignmentResponse alignActiveSites(ActiveSiteAlignmentRequest a
         Page<Motif> motifs = motifService.queryByEcNumber(motifEcNumberFilter, pageNumber);
 
         log.info(String.format("Aligning active sites of %d PDB entries with %d motifs & %d custom motifs.",
-                sourceStructures.size(), motifs.getTotalElements(), customMotifs.size()));
+                sourceStructures.size(), motifs.getTotalElements(), customMotifFileList.size()));
 
         // Align structures with motifs from the database
         while (motifs.hasContent()) {
-            motifs.stream()
```

Comment:
The stream loop here was bloating the length of an alignment request significantly (~30 seconds)